api/scripts/owned_tokens.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+from typing import Optional
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+
+import asyncio
+import logging
+
+import asyncclick as click
+from databases.core import Database
+from sqlalchemy.sql.expression import or_
+from notd.store.retriever import Retriever
+from notd.store.schema import TokenTransfersTable
+from notd.store.schema_conversions import token_transfer_from_row
+
+@click.command()
+@click.option('-a', '--owner-address', 'ownerAddress', required=False, type=str)
+async def ownedTokens(ownerAddress: Optional[str]):
+    database = Database(f'postgresql://{os.environ["DB_USERNAME"]}:{os.environ["DB_PASSWORD"]}@{os.environ["DB_HOST"]}:{os.environ["DB_PORT"]}/{os.environ["DB_NAME"]}')
+    retriever = Retriever(database=database)
+    await database.connect()
+
+    boughtTokens = []
+    soldTokens=[]
+    async with database.transaction():
+        async for tokenTransfer in retrieve_token_transfer(ownerAddress=ownerAddress, retriever=retriever):
+            if tokenTransfer.toAddress == ownerAddress:
+                boughtTokens.append(tokenTransfer.tokenId)
+            if tokenTransfer.fromAddress == ownerAddress:
+                soldTokens.append(tokenTransfer.tokenId)
+
+        uniqueBoughtTokens = set(boughtTokens)
+        uniqueSoldTokens = set(soldTokens)
+        tokensOwned = uniqueBoughtTokens - uniqueSoldTokens
+        for tokenTransfer in tokensOwned:
+            query = TokenTransfersTable.select(TokenTransfersTable.c.tokenId == tokenTransfer)
+            query = query.where(TokenTransfersTable.c.tokenId == tokenTransfer)
+            rows = await database.fetch_all(query=query)
+            for row in rows:
+                print(row[2],row[5])
```

Comment:
whats this?

Reply:
if its debugging, would be better to use `logging.debug` but def give some more context in the log rather than just values e.g. `logging.debug(f'Somethign descriptive: {row[2]}')`

Reply:
I wanted to print out the registry addresses and token id's of the owned tokens

Reply:
cool well make it a nice debug logging line